test/spoom/deadcode/remover_test.rb

```diff
@@ -1027,6 +1027,26 @@ def bar; end
         RB
       end
 
+      def test_deadcode_remover_removes_method_with_rescue
```

Comment:
(nit) Not specific to this PR, but it seems a redundant to start each test name with `test_deadcode_remover_` when we're already inside `DeadCode::RemoverTest`.

Reply:
I'm open to a renaming PR 😂